examples/90time_conv.py

```diff
@@ -3,19 +3,19 @@
 The programmed SpinCore pulse length does not
 match the actual output pulse length. This example
 finds what pulse length should be fed to SpinCore
-in order to get a pulse length with the actual
-desired pulse length.
+in order to get the desired beta.
 """
 import matplotlib.pyplot as plt
 from numpy import r_
 import pyspecdata as psd
 from SpinCore_pp.pulse_length_conv import prog_plen
 
 # the following choice fails dramatically with the old code
-desired_plen = r_[0:50:50j]
-prog_plen = prog_plen(desired_plen)
+desired_beta = r_[10:150:50j]
+prog_plen = prog_plen(desired_beta)
 with psd.figlist_var() as fl:
-    fl.next("Actual vs programmed plen")
-    plt.plot(prog_plen, desired_plen, "o")
-    plt.ylabel(r"actual plen / $\mu$s")
-    plt.xlabel(r"programmed plen / $\mu$s")
+    fl.next(r"Actual vs programmed $\beta$")
```

Comment:
No, it's not the actual versus programmed beta... You want to plot the pulse length versus the desired beta